docs/samples/contact-center/app.js

```diff
@@ -164,7 +164,12 @@ function registerTaskListeners(task) {
   task.on('task:media', (track) => {
     document.getElementById('remote-audio').srcObject = new MediaStream([track]);
   });
-  task.on('task:end', (task) => {
+  task.on('task:end', (wrapupData) => {
+    if (!wrapupData.wrapupRequired) {
+      answerElm.disabled = true;
+      declineElm.disabled = true;
+    }
+    incomingDetailsElm.innerText = '';
```

Comment:
Please add a console.log here indicating that the task has been ended before it was accepted.

Reply:
added